tailwind.config.js

```diff
@@ -2,7 +2,27 @@
 module.exports = {
   content: ["./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        transparent: "transparent",
+        "color-primary-background": "#272B34",
+        "color-secondary-background": "#363a45",
+        "color-primary": "#378DFB",
+        "color-accent": "#e6407a",
+        "color-danger": "#FF4B4C",
+        "color-success": "#63DC93",
+        "color-primary-text": "#FFFFFF",
```

Comment:
As mentioned earlier could possibly change theme to have text-white be the primary one or something similar... basically check if there's other theme options than having text-color-primary-text as a class